include/mme-app/mme_app.h

```diff
@@ -15,6 +15,17 @@
 #include "log.h"
 #include "s1ap_error.h"
 #include "defines.h"
+typedef struct dns_config
+{
+        unsigned int dns_flag;
```

Comment:
dns enabled should be member of this structure ? Want to avoid global individual varaibales. For now only mme top level config variable can be global, nothing else.